MeshLib/IO/MPI_IO/NodePartitionedMeshReader.cpp

```diff
@@ -272,9 +277,10 @@ void NodePartitionedMeshReader::readPropertiesBinary(
     is.open(fname_val.c_str(), std::ios::binary | std::ios::in);
     if (!is)
     {
-        ERR("Could not open file '%s'\n."
-            "\tYou can ignore this warning if the mesh does not contain %s-"
-             "wise property data.", fname_val.c_str(), item_type.data());
+        ERR("Could not open file '{:s}'\n."
+            "\tYou can ignore this warning if the mesh does not contain {:s}-"
+            "wise property data.",
+            fname_val.c_str(), item_type.data());
```

Comment:
`item_type` is already a `string`. Maybe, calling the method `data()` isn't necessary?